Bundled tool atari-hd-image.sh is unusable on OpenBSD because it requires
'mkdosfs' and cannot be patched to use 'newfs_msdos' as a needed option is
missing.

Index: tools/CMakeLists.txt
--- tools/CMakeLists.txt.orig
+++ tools/CMakeLists.txt
@@ -9,7 +9,6 @@ endif(PYTHONINTERP_FOUND)
 
 install(PROGRAMS atari-convert-dir.py DESTINATION ${BINDIR} RENAME atari-convert-dir)
 install(PROGRAMS hatari-prg-args.sh DESTINATION ${BINDIR} RENAME hatari-prg-args)
-install(PROGRAMS atari-hd-image.sh DESTINATION ${BINDIR} RENAME atari-hd-image)
 install(PROGRAMS zip2st.sh DESTINATION ${BINDIR} RENAME zip2st)
 
 if(ENABLE_MAN_PAGES)
@@ -24,12 +23,6 @@ if(ENABLE_MAN_PAGES)
 		COMMAND gzip -c -9 ${CMAKE_CURRENT_SOURCE_DIR}/hatari-prg-args.1 > ${CMAKE_CURRENT_BINARY_DIR}/hatari-prg-args.1.gz
 		DEPENDS hatari-prg-args.1)
 	INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/hatari-prg-args.1.gz DESTINATION ${MANDIR})
-
-	add_custom_target(atari_hd_image_man ALL DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/atari-hd-image.1.gz)
-	add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/atari-hd-image.1.gz
-		COMMAND gzip -c -9 ${CMAKE_CURRENT_SOURCE_DIR}/atari-hd-image.1 > ${CMAKE_CURRENT_BINARY_DIR}/atari-hd-image.1.gz
-		DEPENDS atari-hd-image.1)
-	INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/atari-hd-image.1.gz DESTINATION ${MANDIR})
 
 	add_custom_target(zip2st_man ALL DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/zip2st.1.gz)
 	add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/zip2st.1.gz
